ee/clickhouse/system_status.py

```diff
@@ -111,12 +111,10 @@ def system_status() -> Generator[SystemStatusRow, None, None]:
     total_events_ingested_last_day = sync_execute(
         "SELECT count(*) as b from events WHERE _timestamp >= (NOW() - INTERVAL 1 DAY)"
     )[0][0]
-    dead_letter_queue_ingestion_ratio = dead_letter_queue_events_last_day / total_events_ingested_last_day
+    dead_letter_queue_ingestion_ratio = dead_letter_queue_events_last_day / total_events_ingested_last_day or 1
 
     # if the dead letter queue has as many events today as ingestion, issue an alert
-    dead_letter_queue_events_high = (
-        dead_letter_queue_ingestion_ratio >= 0.5 and dead_letter_queue_events_last_day > 10000
-    )
+    dead_letter_queue_events_high = dead_letter_queue_ingestion_ratio >= 1 and dead_letter_queue_events_last_day > 10000
```

Comment:
hmm... on self-hosted I might want to alert on `dead_letter_queue_events_last_day > 1` or perhaps 10 as they control where the events are coming from & would likely want to know if sending something badly formatted. For our cloud we can probably just use 20% ratio, which I assume is a lot higher compared to how many events currently go to DLQ. Since configuring this seems like an overkill maybe just
```suggestion
    dead_letter_queue_events_high = dead_letter_queue_ingestion_ratio >= 0.2
```